### ISSUE TITLE
Deprecation Warnings In Browser Inspect

### ISSUE THREAD

**Issue by some1ataplace** [jazzband/django-tinymce#417]
tinymce.min.js:9 The following deprecated features are currently enabled, these will be removed in TinyMCE 6.0. See https://www.tiny.cloud/docs/release-notes/6.0-upcoming-changes/ for more information.

Plugins:
- contextmenu
- spellchecker
- textcolor

Settings:
- elements
- force_p_newlines
- mode

**Comment by claudep** [comment]
The fact that TinyMCE removes plugins and replace them by paying functionality doesn't press us to upgrade :cry:

**Comment by some1ataplace** [comment]
So what will happen to this jazzband project then if we don't upgrade? Can't we remove the same stuff just to keep up to date with the newest tinymce version? Otherwise, what happens if one day in the future tinymce says we no longer support version 5? 

I think https://github.com/jazzband/django-tinymce/blob/master/tinymce/static/django_tinymce/init_tinymce.js is somehow automatically appending `mode` and `elements` to TINYMCE_DEFAULT_CONFIG even though inside of my TINYMCE_DEFAULT_CONFIG I did not include `elements` or `mode`. 

When executing these console logs inside init_tinymce.js:
```
      console.log(mce_conf)
      console.log(el.dataset.mceConf)
```

I see `elements` and `mode` shown along with all my other TINYMCE_DEFAULT_CONFIG settings.

The value of elements and mode is:
```
mode:"exact"
elements: "id_body"
```

**Comment by claudep** [comment]
Deprecated settings are gone for some time, so yes, it would be a welcome improvement. Do you plan to work on a patch?

**Comment by some1ataplace** [comment]
Unfortunately I don't really know javascript so I am not sure how to make a patch for it.

**Comment by claudep** [comment]
I would appreciate tests of above pull request.

**Comment by some1ataplace** [comment]
I tested the new code changes you made and it made the warnings go away. Thank you!

**Comment by claudep** [comment]
Was fixed in af4a89c8001b29
